mcnp2cad.cpp

```diff
@@ -799,10 +799,39 @@ entity_collection_t GeometryContext::defineCell(  CellCard& cell,  bool defineEm
       break;
     case CellCard::MBODYFACET:
       {
-        int cellnum = std::abs(token.second) / 10;
-        int facet = std::abs(token.second) - (cellnum*10);
-        std::cerr << "Attempting to define facet " << facet << " of cell " << cellnum << std::endl;
-        throw std::runtime_error( "Macrobody facets are not yet supported by mcnp2cad." );
+        int identifier = -std::abs( token.second );
+        int cellnum = -identifier / 10;
```

Comment:
I know you inherited this name from previous code, but I think this should be interpreted as (macrobody) surface number - probably should change the variable name.